### PR TITLE
Login

### PR DESCRIPTION
**Pull request by fjwillemsen**
Things need to be done:

- [x] implement password forget
   - [ ] mail template
   - [x] send mail
   - [ ] display and handle reset page

### CODE REVIEW

**Comment by barld** [comment]
@fjwillemsen shall i create the implementation of the pasword reset in this pull request?

**Comment by fjwillemsen** [comment]
Yes please @barld!

**Comment by barld** [comment]
Oke in that case keep this request open until i have made a implementation.

File: GoedeDoelenHelpen/ClientApp/src/app/user/login/login.component.ts [hunk 1]
```diff
@@ -27,19 +37,54 @@ export class LoginComponent implements OnInit {
...
   }
 
   onSubmit()  {
     if (this.userLoginGroup.valid) {
       this.authenticationService
         .login({username: this.emailC.value, password: this.passwordC.value})
         .subscribe(
-          success => this.message = 'Je bent ingelogd',
-          err => this.message = 'Je bent niet ingelogd'
+          success => this.loginMessage = 'Je bent ingelogd',
```
Comment (wesselmeijer) [inline GoedeDoelenHelpen/ClientApp/src/app/user/login/login.component.ts]: Ontbreekt hier niet ook de openDialog()?

**Review by wesselmeijer — DISMISSED** [review]

**Review by wesselmeijer — APPROVED** [review]

**Pull request merged**